Day-I-Social-Card/social-card/public/index.html

```diff
@@ -25,6 +25,9 @@
     <noscript>
       You need to enable JavaScript to run this app.
     </noscript>
+    <script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
```

Comment:
These script tags aren't necessary because we are creating a react app by installing react with `yarn install`